### DISCUSSION TITLE
add list of used Github Actions for Action Whitelisting

### DISCUSSION THREAD

**Discussion by KristofKlein** [microsoft/AL-Go#1142] [Q&A] [answered]
Hi,

for my company it would be quite helpful to get an overview of all used Actions. As there is - for GH Enterprise - a setup, that blocks all actions, that are not whitelisted or straight from gh.

While you try to run actions you get a message like "microsoft/AL-Go-Actions/something and 17 others are not allowed." So what are those 17 ? 
Luckily for Al:Go that was now a simple addition like 'microsoft/AL-Go-Actions*'. But now you also make use of other actions in your flows, those I also need to get whitelisted. So maybe that is something that could be mentioned into release notes, if new actions are used. 

so far I found in AL:GO:
- microsoft/AL-Go-Actions*
- microsoft/powerplatform-actions/*
- microsoft/psscriptanalyzer-action

maybe that helps someone

**Comment by freddydk** [comment, marked as answer]
When using AL-Go preview - the actions are in

microsoft/AL-Go (not microsoft/AL-Go-Actions)

But, in general - AL-Go will only use microsoft or GitHub provided actions

- microsoft/*
- actions/*

But Yes, we should add the info to docs somewhere - a list of what actions we depend on.
Will add this.

**Comment by aholstrup1** [comment]
Alternatively, you can always check GitHubs dependency graph for AL-Go-PTE or AL-Go-AppSource:
https://github.com/microsoft/AL-Go-PTE/network/dependencies
https://github.com/microsoft/AL-Go-appsource/network/dependencies

It's a bit verbose but it should show you all the GitHub actions AL-Go depend on and their versions.
